Source/WebInspectorUI/UserInterface/Views/FilterBar.js

```diff
@@ -202,13 +196,8 @@ WI.FilterBar = class FilterBar extends WI.Object
 
     _handleFilterInputEvent(event)
```

Comment:
you can delete this entire function now since it's not used anymore

Reply:
Removed.